### PR TITLE
LUN-423: Upgrade react native to 0.70.7

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -202,13 +191,13 @@ android {
             // Due to a bug inside AGP, we have to explicitly set a dependency
             // between configureNdkBuild* tasks and the preBuild tasks.
             // This can be removed once this is solved: https://issuetracker.google.com/issues/207403732
```
Comment (LeandraH) [inline android/app/build.gradle:193]: Can this comment be removed now?

File: android/app/src/main/jni/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
+
+# Define the library name here.
+project(rndiffapp_appmodules)
```
Comment (LeandraH) [inline android/app/src/main/jni/CMakeLists.txt:4]:
```suggestion
project(lunes_appmodules)
```

**Review by LeandraH — APPROVED** [review]
Very nice, tested on a real iOS device

File: android/app/build.gradle (hunk 1 above)
Comment (sarahsporck) [reply android/app/build.gradle:193]: Can probably be removed when upgrading the android gradle plugin to 7.3+
but has to stay for now

**Review by SteffiStuffel — APPROVED** [review]
Nice work, thank you :pray:

**Pull request merged**